src/actions/images.ts

```
@@ -327,10 +334,17 @@
   return injections;
 };
 
-const generateImages = async (context: MulmoStudioContext, settings?: Record<string, string>, callbacks?: CallbackFunction[]) => {
-  const options = await graphOption(context, settings);
+type ImageOptions = {
+  imageAgents: Record<string, unknown>,
```

ESLint

[error] Replace `,` with `;`

(prettier/prettier)